navigation.yaml

```diff
@@ -1257,6 +1257,9 @@ support:
 
     - title: Community support
       path: /support/community-support
+    
+    - title: Your Offers
```

Comment:
```suggestion
    - title: Your offers
```

Reply:
You can hide this menu with hidden: true

Reply:
But we discussed it being "suppressed" or only visible when you're on the page. The hidden setting hides it at all times.